pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -31,7 +31,11 @@ let
   #
   baseExtensions = self: lib.mapAttrs (_n: lib.recurseIntoAttrs)
     {
-      _1Password.op-vscode = buildVscodeMarketplaceExtension {
+      _1Password = builtins.trace ''_1Password has been replaced with "1Password"'' self."1Password";
+      _2gua = builtins.trace ''_2gua has been replaced with "2gua"'' self."2gua";
+      _4ops = builtins.trace ''_4ops has been replaced with "4ops"'' self."4ops";
```

```suggestion
      _1Password = lib.trace ''_1Password has been replaced with "1Password"'' self."1Password";
      _2gua = lib.trace ''_2gua has been replaced with "2gua"'' self."2gua";
      _4ops = lib.trace ''_4ops has been replaced with "4ops"'' self."4ops";
```
lib.trace adds some extra stuff which we should use https://github.com/NixOS/nixpkgs/blob/master/lib/sources.nix#L110-L131

---

done